rules/postprocess.smk

```diff
@@ -269,6 +269,30 @@ rule make_summary:
         "../scripts/make_summary.py"
 
 
+rule generate_cross_border_flows:
+    input:
+        network=RESULTS
+        + "networks/base_s_{clusters}_{opts}_{sector_opts}_{planning_horizons}.nc",
```

Comment:
if we want to consider existing capacities from conventionals, you should use the add_brownfield function / output